sdk/digitaltwins/digitaltwins/review/digitaltwins.api.md

```diff
@@ -0,0 +1,372 @@
+## API Report File for "@azure/digitaltwins"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { AzureServiceClientOptions } from '@azure/core-arm';
```

Comment:
is this an ARM package rather than a data plane package? If it's data plane, we shouldn't be importing from core-arm

Reply:
It looks like a swagger issue. Let's discuss it on the review meeting.

---
In reply to: [447255188](https://github.com/Azure/azure-sdk-for-js/pull/9787#discussion_r447255188) [](ancestors = 447255188)

Reply:
If this turns out to be a swagger issue, can you let me know so we can fix it for ADT GA?

---
In reply to: [454671521](https://github.com/Azure/azure-sdk-for-js/pull/9787#discussion_r454671521) [](ancestors = 454671521,447255188)

Reply:
@Brian - made a note - I will let you know

---
In reply to: [455377335](https://github.com/Azure/azure-sdk-for-js/pull/9787#discussion_r455377335) [](ancestors = 455377335,454671521,447255188)